### PR TITLE
Add Tf 2.6 inference

### PR DESCRIPTION
**Pull request by hsl89**
### Description
Inference image for tensorflow 2.6.0

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true` or `neuron_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def tfs_model(region, boto_session):
 def python_model_with_requirements(region, boto_session):
     return util.find_or_put_model_data(region,
                                        boto_session,
-                                       "data/python-with-requirements.tar.gz")
```
Comment (jeet4320) [inline test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py:75]: this model supports tf26, right? you need add one more function `python_model_with_requirements_for_greater_than_equal_to_tf26` and use it in tf2.6 test.

File: test/dlc_tests/sanity/test_safety_check.py [hunk 2]
```diff
@@ -148,13 +148,13 @@ def _get_latest_package_version(package):
...
 @pytest.mark.canary("Run safety tests regularly on production images")
 @pytest.mark.skipif(not is_dlc_cicd_context(), reason="Skipping test because it is not running in dlc cicd infra")
-@pytest.mark.skipif(
-    not (is_mainline_context() or (is_canary_context() and is_time_for_canary_safety_scan())),
-    reason=(
-        "Skipping the test to decrease the number of calls to the Safety Check DB. "
-        "Test will be executed in the 'mainline' pipeline and canaries pipeline."
-    )
-)
+# @pytest.mark.skipif(
```
Comment (jeet4320) [inline test/dlc_tests/sanity/test_safety_check.py:151]: things which are added to configs and this commented change will be revert

File: test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py [hunk 3]
```diff
@@ -118,16 +128,37 @@ def test_python_model_with_requirements(boto_session, sagemaker_client,
...
-    input_data = {"x": [1.0, 2.0, 5.0]}
+    input_data = {"instances": [[1.0, 2.0, 5.0]]}
     output_data = util.create_and_invoke_endpoint(boto_session, sagemaker_client,
                                                   sagemaker_runtime_client, model_name,
                                                   python_model_with_requirements, image_uri,
                                                   instance_type, accelerator_type, input_data)
 
-    # python service adds this to tfs response
-    assert output_data["python"] is True
-    assert output_data["pillow"] == "6.0.0"
```
Comment (saimidu) [inline test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py:130]: Why is this removed?

**Review by josephevans — APPROVED** [review]

File: test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py (hunk 3 above)
Comment (hsl89) [reply test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py:130]: because output_data is verified in `def invoke_endpoint` and test code does not make it clear how output_data is modified

File: test/dlc_tests/sanity/test_safety_check.py (hunk 2 above)
Comment (hsl89) [reply test/dlc_tests/sanity/test_safety_check.py:151]: @jeet4320 can you clarify what you mean?

Comment (jeet4320) [reply test/dlc_tests/sanity/test_safety_check.py:151]: before merging this PR, changes added to  dlc_developer_config.toml need to be reverted, which I'll do

**Review by SergTogul — APPROVED** [review]

**Review by SergTogul — APPROVED** [review]
